tomviz/pvextensions/CMakeLists.txt

```diff
@@ -20,8 +20,17 @@ add_paraview_plugin(tomvizExtensions "1.0"
 )
 target_link_libraries(tomvizExtensions PUBLIC pqApplicationComponents)
 
-install(TARGETS tomvizExtensions
-        RUNTIME DESTINATION "bin"
-        LIBRARY DESTINATION "lib"
-        ARCHIVE DESTINATION "lib"
-	COMPONENT "runtime")
+if (NOT APPLE)
+  install(TARGETS tomvizExtensions
+          RUNTIME DESTINATION "bin"
+          LIBRARY DESTINATION "lib"
+          ARCHIVE DESTINATION "lib"
+          COMPONENT "runtime")
+else()
+  install(TARGETS tomvizExtensions
+          RUNTIME DESTINATION "bin"
+          LIBRARY DESTINATION "Applications/tomviz.app/Contents/Libraries"
```

Comment:
So I tried it... it doesn't work.  Apparently only executables can have a `BUNDLE DESTINATION`, it just ignores it for libraries.  (It is also possible that if I could access the `add_library` command and insert the arguments `BUNDLE 1`, it would work... but that isn't possible from `add_paraview_plugin`).  There is also a [cmake bug report](https://gitlab.kitware.com/cmake/cmake/issues/14671) about this saying that it only works for executables and should be extended to all other targets.